src/apply_rules/property_rule.rs

```diff
@@ -47,8 +41,9 @@ enum ObjectType {
     StopPoint,
     StopArea,
 }
+
 impl ObjectType {
```

Comment:
This structure is now duplicated, as well as the `as_str()` implementation (I can see it in `property_rules.rs` and in `complementary_code.rs`). Maybe this need to be kept in `apply_rules/mod.rs`?

Reply:
I need to add physical modes in property rules.

If I keep only one object type, I add physical mode and I decide to add complementary code on an unknown type, the deserializer will tell me that only stop_point.... physical_mode are possible object types, which is wrong because we can't add codes on physical modes.

so each process (property and complementary codes) is applied on differents object types.

Reply:
Ah OK, it makes sense now. So there is really 2 different concepts and it was just coincidental that the same `enum` was used before!